c2corg_ui/templates/waypoint/edit.html

```diff
@@ -35,6 +35,7 @@ <h1 ng-init="id = ${waypoint_id if waypoint_id else 0}" class="text-center">
     <span class="ng-hide">
       {{editCtrl.updateMaxSteps(waypoint.waypoint_type)}} 
       {{editCtrl.waypoint_type = waypoint.waypoint_type}}
+      {{document = waypoint}}
```

Comment:
it is a hack but tried only after seeing that it doesn't work with ng init.
Probably ng init comes to early and has false values, whereas this set can
be modified and updated on the fly.

On Fri, Mar 11, 2016 at 2:45 PM, Alex Saunier <notifications@github.com>
wrote:

> In c2corg_ui/templates/waypoint/edit.html
> <https://github.com/c2corg/v6_ui/pull/224#discussion_r55830010>:
>
> > @@ -35,6 +35,7 @@ <h1 ng-init="id = ${waypoint_id if waypoint_id else 0}" class="text-center">
> >      <span class="ng-hide">
> >        {{editCtrl.updateMaxSteps(waypoint.waypoint_type)}}
> >        {{editCtrl.waypoint_type = waypoint.waypoint_type}}
> > +      {{document = waypoint}}
>
> Does this mean this hidden span is used to set vars?
> Isn't it a bit of a hack?
> Can't you an ng-init for instance?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/c2corg/v6_ui/pull/224/files#r55830010>.
>



-- 

*Adam Gina*JavaScript Developer
adam.gina@camptocamp.com

*Camptocamp SA*EPFL Innovation Park
Bâtiment A
1015 Lausanne

Tel + 41 21 619 10 10
http://www.camptocamp.com